src/ui/Windows/AddEdit_PropertyPage.cpp

```diff
@@ -58,9 +58,6 @@ BOOL CAddEdit_PropertyPage::OnNotify(WPARAM wParam, LPARAM lParam, LRESULT* pRes
   ASSERT(pResult != NULL);
   NMHDR* pNMHDR = (NMHDR*)lParam;
 
-  if (pNMHDR->hwndFrom != m_hWnd && pNMHDR->hwndFrom != ::GetParent(m_hWnd))
```

Comment:
Trying to understand what's happening here and what the issue was. So we're only looking for events coming from either the current window (the edit dialog I assume?) or its parent (the main pwsafe window?), but in this case the event source is something else - what is it? And I wonder why this check was put in place - what events were we trying to filter out and whether they still happen.

Reply:
This method was added in order to set the focus on the Title field in the Basic tab of the Add/Edit entry dialog.

The check I removed prevented Notify events from being propagated from other controls, which is what caused the regression: OnNotify events for the password history table were ignored due to this predicate.

I'm not sure why the predicate was put there in the first place, I suspect in an attempt to avoid false positives for other controls with the same ID as the title field.

In any case, removing this check fixed the problem, and, as far as I could tell by testing, had no other side effects.